bitnami/os-shell/11/debian-11/tags-info.yaml

```diff
@@ -0,0 +1,5 @@
+rolling-tags:
+- "11"
+- "11"
```

Comment:
Is normal this duplicity? I think it's due to another exception given that this asset is not a regular product, but just in case